src/Tasks/InvalidateAllSessionsTask.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace SilverStripe\SessionManager\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\Security\RememberLoginHash;
+use SilverStripe\SessionManager\Models\LoginSession;
+
+class InvalidateAllSessionsTask extends BuildTask
+{
+    /**
+     * @var string
+     */
+    private static $segment = 'InvalidateAllSessions';
```

Comment:
Get rid of the doc blocks and use typed properties - https://www.php.net/manual/en/migration74.new-features.php#migration74.new-features.core.typed-properties.  We use 7.4 as minimum now

Reply:
Wasn't sure we could do that with config - but I guess ultimately if someone puts a non-string in the config it'll just give them an error, which it should. So yup, good point.